include/avif/avif.h

```diff
@@ -1458,6 +1495,11 @@ typedef struct avifEncoder
 #if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
     int qualityGainMap; // changeable encoder setting
 #endif
+
+#if defined(AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM)
+    // Perform extra steps at encoding and decoding to extend AV1 features as bundled additional image items.
```

Comment:
Nit/optional: change "as" to "with" or "using"?

Also, did you mean "AV1 features" or "AVIF features"?

Reply:
I meant "AV1 features". These new "AVIF features" extend the AV1 ones.